reservia.css

```diff
@@ -1,53 +1,675 @@
 @font-face {
```

Comment:
is this file still useful ? i remember you removed it from the html, so don't' hesitate to remove it totally from your source code. dead code is useless and take space for nothing and might add confusion to your project.